meta/w3scout/contao-dompdf-bundle/en.yml

```diff
@@ -5,4 +5,4 @@ de:
     keywords:
     - contao
     - bundle
-    - pdf
\ No newline at end of file
+    - pdf
```

Comment:
Same here :)

Reply:
Not sure, I understand the problem. But hoping, it´s fixed now?!